core/src/test/java/org/hisp/dhis/android/core/trackedentity/search/TrackedEntityInstanceQueryOnlineHelperShould.kt

```diff
@@ -85,4 +86,33 @@ class TrackedEntityInstanceQueryOnlineHelperShould {
         assertThat(onlineQueries.size).isEqualTo(1)
         assertThat(onlineQueries[0].attributeFilter.size).isEqualTo(1)
     }
+
+    @Test
+    fun to_API_filter_format() {
+        // List of filters
+        val list = listOf(
+            "nom,app",
+            "nom:app",
+            "nom;app"
+        )
+
+        val expectedList = listOf(
+            "filterItem1:in:nom/,app/;nom/:app/;nom/;app",
```

Comment:
For `in` operator, the expected string should be `nom/,app;nom/:app;nom/;app`. The char `;` separates the items in the list and must not be scaped.